front/src/views/Home.vue

```diff
@@ -2,31 +2,27 @@
     <div class="home">
         <v-container>
             <v-row dense>
-                <v-col cols="12">
+                <v-col cols="12" v-if="user">
```

Comment:
maybe put the `v-if` on `v-container` ?
If no user we will display a totally different page
wdyt ?

Reply:
??